CHANGELOG.md

```diff
@@ -61,6 +61,7 @@ To learn more about active deprecations, we recommend checking [GitHub Discussio
 - **Pulsar Scaler**: Improve error messages for unsuccessful connections ([#4563](https://github.com/kedacore/keda/issues/4563))
 - **Security:** Enable secret scanning in GitHub repo
 - **RabbitMQ Scaler**: Add support for `unsafeSsl` in trigger metadata ([#4448](https://github.com/kedacore/keda/issues/4448))
+- **RabbitMQ Scaler**: Add support for `avg_rate` in trigger metadata ([#4507](https://github.com/kedacore/keda/issues/4507))
```

Comment:
This is not really the name of the field, let's provide a better reference here so that it's clear what everyone can use

Reply:
The field `avg_rate` it's referring to is the field exposed by RabbitMQ in the JSON repsonse. See [HTTP API Stats
](https://rawcdn.githack.com/rabbitmq/rabbitmq-server/main/deps/rabbitmq_management/priv/www/api/index.html). 

I see your point that the phrasing is ambiguous, I'll rewort it to make it clearer.